lib/api/src/Cardano/Wallet/Api/Http/Server/Error.hs

```diff
@@ -626,18 +628,35 @@ instance IsServerError ErrRemoveTx where
 instance IsServerError ErrPostTx where
     toServerError = \case
         ErrPostTxValidationError err ->
-            apiError err500 CreatedInvalidTransaction $ mconcat
-                [ "The submitted transaction was rejected by the local "
-                , "node. Here's an error message that may help with "
-                , "debugging:\n", err
-                ]
+            apiError err500 CreatedInvalidTransaction
+                $ mconcat
+                    [ "The submitted transaction was rejected by the local "
+                    , "node. Here's an error message that may help with "
+                    , "debugging:\n"
+                    , err
+                    ]
         ErrPostTxMempoolFull ->
-            apiError err425
-            {errBody = "Mempool is full, please try resubmitting again later."}
-                MempoolIsFull $ mconcat
-                [ "The submitted transaction was rejected by the Cardano node "
-                , "because its mempool was full."
-                ]
+            apiError
+                err425
+                    { errBody = "Mempool is full, please try resubmitting again later."
+                    }
+                MempoolIsFull
+                $ mconcat
+                    [ "The submitted transaction was rejected by the Cardano node "
+                    , "because its mempool was full."
+                    ]
+        e@(ErrPostTxEraUnsupported unsupported) ->
+            apiError err403 error' $ toText e
+          where
+            error' =
+                UnsupportedEra
+                    $ ApiErrorUnsupportedEra
+                        (toApiEra unsupported)
+                    $ Set.fromList
+                        ( toApiEra
+                            . Write.toAnyCardanoEra
+                            <$> [minBound .. maxBound]
+                        )
```

Comment:
IMHO it's not very easy for readers to see that:
- this is constructing a record with two fields (due to different indent levels)
- the second field is the _set of all supported recent eras_

**Suggestion**:

Rewrite this so that fields are named explicitly:

```suggestion
            flip (apiError err403) (toText e) $
                UnsupportedEra
                ApiErrorUnsupportedEra
                    { unsupportedEra = toApiEra unsupported
                    , supportedEras =
                        Set.fromList $
                        map (toApiEra . Write.toAnyCardanoEra)
                            [minBound .. maxBound]
                    }
```

This also saves a bit of space, and avoids having to define `error'`.